src/components/Referrals/referralsHelper.js

```diff
@@ -1,15 +1,16 @@
 import {
   formatAmount,
-  USD_DECIMALS,
   bigNumberify,
+  isAddressZero,
+  USD_DECIMALS,
   MAX_REFERRAL_CODE_LENGTH,
   ARBITRUM,
   AVALANCHE,
-  isAddressZero,
 } from "../../Helpers";
 import { encodeReferralCode, getReferralCodeOwner } from "../../Api/referrals";
 
 export const REFERRAL_CODE_REGEX = /^\w+$/; // only number, string and underscore is allowed
+export const REGEX_VERIFY_BYTES32 = new RegExp(/0x[0-9a-f]{64}/);
```

Comment:
- i guess `new RegExp` is not needed here, it's already regexp because of leading and trailing slashes
- better to add `^` at the beginning and `$` in the end otherwise strings like "!@&#^%!@# 0xfac123.. !@#!@#" would be valid

Reply:
Sure! I have updated it to `/^0x[0-9a-f]{64}$/`